.github/workflows/pytest.yml

```diff
@@ -0,0 +1,45 @@
+name: pytest (RHEL 8, RHEL 9)
```

Comment:
I'd keep only "pytest", as you get the actual tested distros by looking at the matrix jobs